build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -18,19 +18,24 @@ public struct VerInfo
             public string Release;
             public string BuildMajor;
             public string BuildMinor;
+            public string CommitCountString;
 
-            public VerInfo(int major, int minor, int patch, string release, string buildMajor, string buildMinor)
+            public VerInfo(int major, int minor, int patch, string release, string buildMajor, string buildMinor, string commitCountString)
             {
                 Major = major;
                 Minor = minor;
                 Patch = patch;
                 Release = release;
                 BuildMajor = buildMajor;
                 BuildMinor = buildMinor;
+                CommitCountString = commitCountString;
             }
 
             public string WithoutSuffix => $"{Major}.{Minor}.{Patch}";
 
+            public string MsiBuildMajor => string.IsNullOrEmpty(BuildMajor) ? BuildMajor : CommitCountString;
+            public string MsiBuildMinor => string.IsNullOrEmpty(BuildMinor) ? BuildMinor : "00";
```

Comment:
I'm not sure I understand the name here: `MsiBuildMajor`?  This is the version of the Msi?